Prepare documentation suite (#98)
Jelly-RDF/pyjelly: docs/overview.md

## What is Jelly and what is `pyjelly`?

> [Jelly]({{ proto_link() }}) is a serialization and streaming protocol for RDF. It enables fast, compact, and flexible transmission of RDF data using Protocol Buffers, supporting both flat and structured streams of triples, quads, graphs, and datasets. Jelly is designed to work well in both batch and real-time settings, including use over files, sockets, or stream processing systems like Kafka or gRPC.

Reviewer: ```suggestion
[Jelly]({{ proto_link() }}) is a serialization format and streaming protocol for RDF knowledge graphs. It enables fast, compact, and flexible transmission of RDF data with Protobuf, supporting both flat and structured streams of triples, quads, graphs, and datasets. Jelly is designed to work well in both batch and real-time settings, including use over files, sockets, or stream processing systems like Kafka or gRPC.
```

**`pyjelly`** is a Python implementation of the Jelly protocol. It provides:

* Full support for reading and writing Jelly-encoded RDF data

> * Seamless integration with [RDFLib](https://rdflib.readthedocs.io/) via plugin entry points

Reviewer: ```suggestion
* Seamless integration with [RDFLib](https://rdflib.readthedocs.io/) (*"works just like Turtle"*)
```

> * Support for all stream types defined in the Jelly specification

Reviewer: ```suggestion
* Support for all Jelly stream types
```
